tests/php/modules/shortcodes/test_class.kickstarter.php

```diff
@@ -44,6 +44,7 @@ public function test_shortcodes_kickstarter_invalid_url() {
 	 * @since 4.5.0
 	 */
 	public function test_shortcodes_kickstarter_image() {
+		$this->markTestSkipped();
```

Comment:
I think this is fine to remove now.  Tests are passing in other PRs

Reply:
lets skip it for now.